tox.ini

```diff
@@ -10,13 +10,17 @@ commands = test_plugin --standalone plugin.py
 commands = test_plugin --standalone plugin.py --regenerate
 
 [testenv:package]
-commands = build_plugin plugin.py . {posargs}
+deps = hansken-extraction-plugin==0.8.3
+commands = build_plugin . {posargs}
 
 [testenv:integration-test]
+deps = hansken-extraction-plugin==0.8.3
 commands =
-  build_plugin plugin.py . plugin-to-test {posargs}
+  build_plugin . plugin-to-test {posargs}
   test_plugin --docker plugin-to-test
 
 [testenv:upgrade]
 deps = pip-tools
-commands = pip-compile --upgrade --no-header --no-emit-options --strip-extras --output-file requirements.txt requirements.in
+commands =
+  pip-compile --upgrade --no-header --no-emit-options --strip-extras --output-file requirements.txt requirements.in
+  python -c 'import shutil, os; os.path.exists(".tox") and shutil.rmtree(".tox")'
```

Comment:
Is that the way? Won't that remove half of the import path of the running interpreter during `tox -e upgrade`? :thinking:

Reply:
Tox does not seems to automatically update its virtualenvs if requirements.txt is updated (at least not on our Ubuntu machines). Twice we had to spend a lot of time finding out why things were not working, only to find out that the tox virtual-env was not in updated.  Removing the entire .tox directory forces tox to create a fresh virtualenv next time it is invoked, with our-up-to-date dependencies. The removal here during the execution of `tox -e upgrade` seems to work fine 😅.

If you have another recommendation, please let me know!